src/lavatory/commands/stats.py

```diff
@@ -19,9 +19,14 @@
 def stats(ctx, repo):
     """Get statistics of repos."""
     LOG.debug('Passed args: %s, %s.', ctx, repo)
-    storage, keys = get_artifactory_info(repo_names=repo, repo_type='any')
 
-    for repository in keys:
+    storage = get_storage(repo_names=repo, repo_type='any')
+    if not storage:
+        click.echo('User does not have "Admin Privileges" to generate statistics.')
```

Comment:
Go with INFO level message like other admin related messages.
```suggestion
        LOG.info('User does not have "Admin Privileges" to generate statistics.')
```

Reply:
Updated as suggested.